forum/qa-content/qa-page.js

```diff
@@ -177,6 +177,58 @@ function qa_ajax_error()
 }
 
 
+/**
+ *	Feature: check if browser supports copying to clipboard.
+ *	If true, return function, which can copy highlighted content to clipboard on button click - then user can paste it wherever he wants into.
+ *	Otherwise return null.
+ */
+window.initializeCopyToClipboard = ( function() {
+	// check if browser supports 'select()' and 'copy' commands
+	var isClipboardSupported = ( window.getSelection && document.queryCommandSupported( 'copy' ) && navigator.userAgent.indexOf( 'Firefox' ) === -1 );

```

Comment:
way too long line; makes it unreadable. 100 or at most 120 chars per line, ideally it would be checked with some code linter.